src/integTest/java/org/sonatype/gradle/plugins/scan/ScanPluginIntegrationTestBase.java

```diff
@@ -414,6 +416,36 @@ public void testAuditTask_AndroidMultipleFlavors_OssIndex() {
     assertThat(result.task(":ossIndexAudit").getOutcome()).isEqualTo(SUCCESS);
   }
 
+  @Test
+  public void testAuditTask_PrintBanner_True() throws IOException {
+    testAuditTask_PrintBanner("print_banner_true.gradle", true);
```

Comment:
Since the banner is present by default, maybe instead of this test adding the assert for the banner is good enough using the control file.

Reply:
> Since the banner is present by default, maybe instead of this test adding the assert for the banner is good enough using the control file.

Test removed and assertions added in 22c5d6d2